hipify_config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Why is this added into hipify repo?

Is the intention to add a sample file, then I would suggest to put into README.md or documentation.

Reply:
@bmedishe I agree, let's remove this file and add the sample contents to the README.md. Also, in the example, `project_directory` should be a relative path, preferably relative to the location of the hipify_config.json file to make it most intuitive to the user. Lmk if that doesn't work.